src/charm.py

```diff
@@ -147,7 +147,7 @@ def _restart(self, _) -> None:
     def init_server(self):
         """Calls startup functions for server start.
 
-        Sets myid, opts env_var, initial servers in dynamic properties,
+        Sets myid, server_jvmflgas env_var, initial servers in dynamic properties,
```

Comment:
what is `jvmflgas`? I think it might be better as `server_jvm_flags`, but still don't know what a jvm flag is

Reply:
I kept the name the same as the env-var `SERVER_JVMFLAGS` expected by ZooKeeper.
FWIW, JVM flags are like run-time arguments for the Java machine, Mehdi has an example of more extensive JVM options with the opensearch operator here: https://github.com/canonical/opensearch-operator/blob/unit-tests_init-charm-tls/tests/unit/resources/config/jvm.options